.github/DISCUSSION_TEMPLATE/question.yml

```
@@ -0,0 +1,63 @@
+title: "Summary"
+labels: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        We are glad to have you build on Aptos with us!
+        Please choose some labels and fill out the questions below.
+  - type: textarea
+    id: question
+    attributes:
+      label: Describe your question in detail
+    validations:
+      required: true
+  - type: textarea
+    id: error
+    attributes:
+      label: What error are you getting?
+    validations:
+      required: true
+  - type: textarea
+    id: attempt
+    attributes:
+      label: What have you tried or how can we reproduce?
+      description: |
+        Code snippet is preferred over screenshots
+    validations:
+      required: true
+  - type: dropdown
+    id: os
+    attributes:
+      label: What operating system are you using?
+      options:
+        - Windows
+        - Mac OS
+        - Linux (Ubuntu, Fedora, Windows WSL, etc.)
+        - N/A
+    validations:
+      required: true
+  - type: dropdown
+    id: tool
+    attributes:
+      label: What tool are you using?
+      options:
+        - TypesScript SDK
+        - Python SDK
+        - Rust SDK
+        - Unity SDK
+        - Aptos CLI
+        - Rest API
+        - N/A
+    validations:
+      required: true
+  - type: textarea
+    id: env
+    attributes:
+      label: Describe your environment or tooling in detail
+      value: |
+        - OS: Windows 10
+        - TS-SDK: 1.12.2
+        - Aptos CLI: 3.1.0
+        - AptosFramework revision: mainnet
+        - Anything else you think is relevant
```

Let's not fill out the value automatically, it'll probably be always defaulted.
```suggestion
      label: Describe your environment or tooling in detail
      description: |
        Operating system, computer architecture, SDK or tool version, AptosFramework version, anything else you feel is relevant
```